stdlib/3/http/cookiejar.pyi

```diff
@@ -28,6 +26,7 @@ class CookieJar(Iterable['Cookie']):
               name: str = ...) -> None: ...
     def clear_session_cookies(self) -> None: ...
     def __iter__(self) -> Iterator['Cookie']: ...
+    def __len__(self) -> int: ...
```

Comment:
This is needed to make requests.cookies.RequestsCookieJar concrete. It's defined here: https://github.com/python/cpython/blob/master/Lib/http/cookiejar.py#L1735.